templates/common/_base/units/machine-config-daemon-firstboot.service.yaml

```diff
@@ -14,7 +14,7 @@ contents: |
   Type=oneshot
   RemainAfterExit=yes
   # Disable existing repos (if any) so that OS extensions would use embedded RPMs only
-  ExecStartPre=-/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo"
+  ExecStartPre=-/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo; sed -i 's/enabled=0/enabled=1/' /etc/yum.repos.d/rhel-coreos-8-extensions.repo"
```

Comment:
Hmmmm.  I wonder if we really still need this thing at all.  We shouldn't have any enabled repos by default, no?

Ahhh I see this came in with https://github.com/openshift/machine-config-operator/commit/552f6d5fb5c62aa5b6da01ee8266731f33af14e1 for OKD (FCOS) which does have repos.

Hmm.  I'm thinking that if they wanted the repos disabled, they should do this process in their builds, and not here.

As is this line is going to actively fight anyone trying to add repos to RHCOS day 0, which I think is a somewhat reasonable thing to do.

Anyways, OK as is - clearly the tech debt here isn't new to this change.